examples/using-gatsby-image/src/components/image-gallery.js

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import Img from "gatsby-image"
+import numeral from "numeral"
+
+import { options, scale } from "../utils/typography"
+import presets from "../utils/presets"
+
+const UnsplashMasonry = edges => (
+  <div
+    css={{
+      background: `white`,
+      margin: `0 -`,
+      [presets.Tablet]: {
+        margin: `0 0 0 calc(-${presets.offset}vw - ${presets.gutter.tablet}px)`,
+        padding: 100,
+        paddingRight: 0,
+      },
+      [presets.Desktop]: {
+        margin: `0 0 0 calc(-${presets.offset}vw - ${
+          presets.gutter.desktop
+        }px)`,
+      },
+    }}
+  >
+    <div
+      css={{
+        columnCount: 1,
+        columnGap: 0,
+        margin: `0 auto`,
+        [presets.Mobile]: {
+          columnCount: 2,
+        },
+        [presets.Tablet]: {
+          columnCount: 3,
+        },
+        "@media screen and (min-width: 1400px)": {
+          columnCount: 4,
+        },
```

Comment:
For consistency, can we put everything into `presets` or pull everything out of it?

Reply:
Oh, missed that one (and probably more) … 👍

Reply:
Done!